frontend/src/components/common/CreateResourceButton.tsx

```diff
@@ -36,6 +36,7 @@ export function CreateResourceButton(props: CreateResourceButtonProps) {
         errorMessage={errorMessage}
         onEditorChanged={() => setErrorMessage('')}
         title={t('translation|Create {{ name }}', { name })}
+        aria-label={t('translation|Create resource button')}
```

Comment:
- Probably should be fixed inside EditorDialog to use the title.
- Using the same name as the title would be better, because it's more correct and we already have translations for it.